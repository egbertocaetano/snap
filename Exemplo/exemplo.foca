int main()
{	

	float b = 0.0;
	int a = b * 4;
	boolean c = TRUE;
	c = 5.0<3;
}
