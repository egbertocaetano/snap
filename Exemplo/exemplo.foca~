int main()
{	

	int a=6;
	int b=2;
	int c;
	c=a+b;
}
