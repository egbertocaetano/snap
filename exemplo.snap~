int main()
{
	int 4 + 6;
	2 - 3;
	1 + 1;

	a = 7 + 8;
}
