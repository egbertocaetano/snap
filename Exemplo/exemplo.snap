int main()
{	
	int tests;
	float b = 0.0;
	int a = b + 4;
	boolean c = TRUE;
	c = 5.0<3;
}
